docs/reference/commands/cluster.rst

```diff
@@ -0,0 +1,210 @@
+===========
+``cluster``
+===========
+
+Manage the MicroCeph cluster
+
+Usage:
+
+.. code-block:: none
+
+   microceph cluster [options]
+   microceph cluster [command]
+
+Available commands:
+
+.. code-block:: none
+
+  add         Generates a token for a new server
+  bootstrap   Sets up a new cluster
+  config      Manage Ceph Cluster configs
+  join        Joins an existing cluster
+  list        List servers in the cluster
+  migrate     Migrate automatic services from one node to another
+  remove      Removes a server from the cluster
+  sql         Runs a SQL query against the cluster database
+
+
+Global options:
+
+.. code-block:: none
+
+   -d, --debug       Show all debug messages
+   -h, --help        Print help
+       --state-dir   Path to store state information
+   -v, --verbose     Show all information messages
+       --version     Print version number
+
+``add``
+-------
+
+Generates a token for a new server
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster add <NAME> [flags]
+
+
+``bootstrap``
+-------------
+
+Sets up a new cluster
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster bootstrap [flags]
```

Comment:
We'll need to include what the s/flags/options are. The existing 'remove' sub-command on the 'disk' command page includes:

```
Options:

.. code-block:: none

   --bypass-safety-checks               Bypass safety checks
   ...
   ...
```

Reply:
In this case this would be global flags. I'm not sure we want to repeat them?

Reply:
Oh, if they are available as global flags then no, let's not repeat.